apps/measures/templates/apps_measures/measures_detail_page.html

```diff
@@ -0,0 +1,97 @@
+{% extends "base.html" %}
+{% load wagtailcore_tags wagtailimages_tags i18n %}
+
+{% block content %}
+<div id="layout-grid__area--maincontent">
+    <h1>{{ page.page_title }}</h1>
```

Comment:
check styleguide or our code for required classes for headers, think its just 'title' same for all elements on page, img, p included